src/Controller/Controller_System_Report.php

```diff
@@ -63,24 +51,39 @@ public function init() {
 	}
 
 	/**
-	 * inserted data below the MB String row in the PHP section
+	 * Apply any filters needed for the settings page
+	 *
+	 * @since 5.2.0
+	 *
+	 * @return void
 	 */
 	public function add_filters() {
-		add_filter( 'gform_system_report', function( $report ) {
+		add_filter( 'gform_system_report', [ $this, 'system_report' ] );
+	}
+
+	/**
+	 * Include the add-on table in the PHP Server Environment system report.
+	 *
+	 * @since 5.2.0
+	 *
+	 * @param array $system_report
+	 *
+	 * @return array
+	 */
+	public function system_report( $system_report ) {
 
-			if ( isset( $report[2]['tables'][1]['items'] ) && is_array( $report[2]['tables'][1]['items'] ) ) {
-				$insert_val[] = [
-					'label'        => 'alow_url_fopen',
-					'label_export' => 'alow_url_fopen',
-					'value'        => $this->check_allow_url( $this->data ),
-					'value_export' => $this->check_allow_url( $this->data ),
-				];
+		if ( isset( $system_report[2]['tables'][1]['items'] ) && is_array( $system_report[2]['tables'][1]['items'] ) ) {
+			$insert_val[] = [
+				'label'        => 'alow_url_fopen',
+				'label_export' => 'alow_url_fopen',
+				'value'        => $this->check_allow_url( $this->data ),
+				'value_export' => $this->check_allow_url( $this->data ),
```

Comment:
Assign results of `$this->check_allow_url( $this->data )` to a variable. Saves calling the method twice.